engines/stark/ui/world/dialogpanel.cpp

```diff
@@ -174,25 +181,44 @@ void DialogPanel::updateDialogOptions() {
 	clearOptions();
 
 	_firstVisibleOption = 0;
+	_focusedOption = 0;
 	Common::Array<DialogPlayer::Option> options = StarkDialogPlayer->listOptions();
 
 	for (uint i = 0; i < options.size(); i++) {
 		_options.push_back(new ClickText(options[i]._caption, _aprilColor));
 	}
+
+	if (!_options.empty()) {
+		_options[_focusedOption]->setActive();
+	}
 }
 
 void DialogPanel::onMouseMove(const Common::Point &pos) {
+	static Common::Point prevPos;
+
+	// Mouse Move event will always exists. Specifically handled here.
+	if (pos == prevPos && !_options.empty()) {
```

Comment:
It seems that the mouse move event will always be there, even when the mouse is not moving at all, yet the dialog panel requires that players can use the up or down key to select options even when the mouse hovers on an option. I am not sure how `onMouseMove()` depends on the event so I only tackle this special case here.